consumer.go

```diff
@@ -277,7 +282,7 @@ func (child *partitionConsumer) dispatcher() {
 				select {
 				case <-child.dying:
 					close(child.trigger)
-				case <-time.After(10 * time.Second):
+				case <-time.After(child.conf.Consumer.Retry.Backoff):
```

Comment:
With this loop fixed I think it makes much more sense to drop the default to 200ms or something more reasonable - the high value was my subconscious trying to point out that the delay was in the wrong place anyways.